test/test_jit.py

```diff
@@ -3500,6 +3500,12 @@ def bad_negative_index():
         self.checkScriptRaisesRegex(bad_negative_index, (), IndexError,
                                     "list index out of range")
 
+    def test_tensor_len(self):
+        def func(x):
+            return len(x) == 4
+
+        self.checkScript(func, [torch.ones(4, 5, 6)])
```

Comment:
nit: func(x) is going to return false, so if len(x) were implemented to always return 0, we would not catch such a bug. Could you add another line that tests `torch.ones(4)` as an input?

Reply:
shouldn't this `func` return true? Since 4 is the size of the first dimension.

Reply:
Oh oops I forgot that is len's behavior. Carry on.